### ISSUE TITLE
cannot create an environmental sample collection

### ISSUE THREAD

**Issue by elenimijalis** [TAMU-CPT/lims-frontend#9]
The field to add individual env samples is just a text field.

**Comment by elenimijalis** [comment]
also need to add storage to this

**Comment by hexylena** [comment]
Yup, totally not implemented at all yet. I didn't have a good idea of how people would want to select env samples.

- If select by location => map and JS craziness to select locations
- if select by metdata (isolator / date / sample type) => table.

**Comment by elenimijalis** [comment]
Yeah, maybe there should be a table with checkmarks on the rows. The user can choose existing samples. or there can be a popup for adding new samples identical to the existing way to add an env sample. When the popup closes, the sample will be added to the table with and automatically checked.

**Comment by hexylena** [comment]
yeah, table is probably the easiest way to go.

perhaps we can shift a popup with auto-add functionality to the next version of LIMS.

**Comment by elenimijalis** [comment]
shift a popup with auto-add functionality?

**Comment by hexylena** [comment]
move the feature "or there can be a popup for adding new samples" to a later release of LIMS.
